<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dexter's WorkShop - DMX Console</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="4" column="1">
     <widget class="QPushButton" name="stop_a">
      <property name="styleSheet">
       <string notr="true">background: red;</string>
      </property>
      <property name="text">
       <string>STOP AQUISITION</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="l_temp">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>80</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: green;
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>TEMP</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn_exit">
      <property name="styleSheet">
       <string notr="true">background: red;</string>
      </property>
      <property name="text">
       <string>EXIT APPLICATION</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="l_hum">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>80</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: green;
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>H%</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="start_a">
      <property name="styleSheet">
       <string notr="true">background: green;</string>
      </property>
      <property name="text">
       <string>START AQUISITION</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLCDNumber" name="hum">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLCDNumber" name="temp">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
